File: python-sdk/src/formanceapi/models/shared/runworkflowresponse.py
import dataclasses
from ..shared import workflowoccurrence as shared_workflowoccurrence
from dataclasses_json import dataclass_json
from formanceapi import utils


@dataclass_json
@dataclasses.dataclass
class RunWorkflowResponse:
    data: shared_workflowoccurrence.WorkflowOccurrence = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('data') }})
    

File: ts-sdk/src/sdk/models/shared/runworkflowresponse.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";
import { WorkflowOccurrence } from "./workflowoccurrence";


export class RunWorkflowResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=data" })
  data: WorkflowOccurrence;
}